eng/pipelines/merge-docs-main-to-live.yml

```diff
@@ -0,0 +1,87 @@
+parameters:
+- name: WhatIfPreference
+  type: boolean
+  default: false
+
+- name: DocsRepos
+  type: object
+  default:
+    - RepoName: azure-docs-sdk-java
+      RepoOwner: Azure
+      AllowlistPath: eng/pipelines/templates/scripts/docsms-allowlist/java-allowlist.txt
```

Comment:
If we read the allowlist in docs repo, we do not need to bother on the changes of the allowlist. It is all related to docs repo. I think it does not need to request review from Eng team.

Reply:
This sounds like a configuration file unique to the docs repo so with having it there and this can just be a path to where we read that configuration file in the docs repo.